Dockerfile/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:18.04
+MAINTAINER Jan Malec (jan.malec@gmail.com)
+
+RUN apt update
+RUN apt install -y gfortran python3 python3-setuptools\
+    python3-numpy python3-dev unzip ca-certificates
+
+COPY sandy-develop.zip /
```

Comment:
Instead of using `COPY` and a zip file, install `git` and run a clone command. :wink:

Reply:
Sorry for the late response, I will change the Dockerfile and push the update!